src/YamlValidator.Tests/ValidatorTest.cs

```diff
@@ -9,38 +9,49 @@ namespace Persistence.Tests.YamlValidator;
 [TestClass]
 public class ValidatorTest
 {
-    private const string _validPath = @".\_TestData\ValidYaml";
-    private const string _invalidPath = @".\_TestData\InvalidYaml";
-    private const string _schemaPath = @"..\YamlValidator\schema\pa.yaml-schema.json";
+
+    private static readonly string _validPath = Path.Combine(
+        [".", "_TestData", "ValidYaml"]) + Path.DirectorySeparatorChar;
```

Comment:
This overload of `Path.Combine(...)` takes a `params` argument, so no need to create an array yourself here - you can remove the `[` and `]` and the compiler handles the array creation:
```suggestion
    private static readonly string _validPath = Path.Combine(".", "_TestData", "ValidYaml") + Path.DirectorySeparatorChar;
```